GCRCatalogs/catalog_configs/dc2_object_run1.2p.yaml

```diff
@@ -1,8 +1,8 @@
 subclass_name: dc2_object.DC2ObjectCatalog
```

Comment:
Use the `alias` feature instead of repeating the content of the yaml file.

Suggestion: change this whole config to just:
```yaml
alias: dc2_object_run1.2p_v4
included_by_default: true
```